Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.56.1 as chef
```

Comment:
I'd love some comments on this file. I can't really interpret what's going on here.